### ISSUE TITLE
Double indentation on wrapped function arguments

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1261]
```diff
-    if (m_Preview.GetExpanded())
-    {
-        m_EventSearchString = EditorGUI.TextField(searchRect, m_EventSearchString, Styles.toolbarSearchField);
-        if (GUILayout.Button(
-            GUIContent.none,
-            m_EventSearchString == string.Empty ? Styles.toolbarSearchFieldCancelEmpty : Styles.toolbarSearchFieldCancel))
+    if (m_Preview.GetExpanded())
+    {
+        m_EventSearchString = EditorGUI.TextField(searchRect, m_EventSearchString, Styles.toolbarSearchField);
+        if (GUILayout.Button(
+                GUIContent.none,
+                m_EventSearchString == string.Empty ? Styles.toolbarSearchFieldCancelEmpty : Styles.toolbarSearchFieldCancel))
```

```diff
bool success = GenerateSecondaryUVSet(
-        &mesh.vertices[0].x, mesh.vertices.size(),
-        &triUV[0].x, &triList[0], triSrcPoly.size() ? &triSrcPoly[0] : 0, triCount,
-        &outUV[0].x, param, errorBuffer, bufferSize
-    );
+            &mesh.vertices[0].x, mesh.vertices.size(),
+            &triUV[0].x, &triList[0], triSrcPoly.size() ? &triSrcPoly[0] : 0, triCount,
+            &outUV[0].x, param, errorBuffer, bufferSize
+            );
```

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60013-UNI-13955.cs" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60013-UNI-13955.cs"
index 940aa89..e5e7e3c 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60013-UNI-13955.cs"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60013-UNI-13955.cs"
@@ -1,16 +1,16 @@
 if (m_Preview.GetExpanded())
 {
     m_EventSearchString = EditorGUI.TextField(searchRect, m_EventSearchString, Styles.toolbarSearchField);
     if (GUILayout.Button(
-        GUIContent.none,
-        m_EventSearchString == string.Empty ? Styles.toolbarSearchFieldCancelEmpty : Styles.toolbarSearchFieldCancel))
+            GUIContent.none,
+            m_EventSearchString == string.Empty ? Styles.toolbarSearchFieldCancelEmpty : Styles.toolbarSearchFieldCancel))
     {
     }
 }
 
 // The closing parenthesis is being indented twice.
 bool success = GenerateSecondaryUVSet(
-    &mesh.vertices[0].x, mesh.vertices.size(),
-    &triUV[0].x, &triList[0], triSrcPoly.size() ? &triSrcPoly[0] : 0, triCount,
-    &outUV[0].x, param, errorBuffer, bufferSize
-);
+        &mesh.vertices[0].x, mesh.vertices.size(),
+        &triUV[0].x, &triList[0], triSrcPoly.size() ? &triSrcPoly[0] : 0, triCount,
+        &outUV[0].x, param, errorBuffer, bufferSize
+        );
```

Debug support files:
---
[60013-UNI-13955.cs](https://gist.githubusercontent.com/mihaipopescu/59711b56adbbcd82cb643bc0ad5c8c21/raw/940aa89bf1e120f3a7d1c04d038d83d79bf5571a/60013-UNI-13955.cs)
[60013-UNI-13955.cs.log](https://gist.githubusercontent.com/mihaipopescu/59711b56adbbcd82cb643bc0ad5c8c21/raw/7bac43389c0dd6910a8b08cfbe38146ff5feb95a/60013-UNI-13955.cs.log)
[60013-UNI-13955.cs.unc](https://gist.githubusercontent.com/mihaipopescu/59711b56adbbcd82cb643bc0ad5c8c21/raw/5ad8de950e7005c518ea88b7330cb9e032da0a3a/60013-UNI-13955.cs.unc)
[Uncrustify.CSharp.cfg](https://gist.githubusercontent.com/mihaipopescu/59711b56adbbcd82cb643bc0ad5c8c21/raw/1d064c1fabb76ce328dd015ed40023207de20ca1/Uncrustify.CSharp.cfg)
[UNI-13955.cs](https://gist.githubusercontent.com/mihaipopescu/59711b56adbbcd82cb643bc0ad5c8c21/raw/940aa89bf1e120f3a7d1c04d038d83d79bf5571a/UNI-13955.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
The first example is Ok.
If after `GUILayout.Button(` a param would follow the next line would be indented to the first char of that first param: 
```cpp
    (GUILayout.Button(int p
                      ^
```
In this case there is no param and the continuation indent kicks in, that one is set to zero so the value of `indent_columns` is used. First it is indented with `indent_columns` than the continuation for the first paren 
and then the one for the second
```cpp
    if (GUILayout.Button(
    ^   ^   ^
            GUIContent.none,
```

**Comment by mihaipopescu** [comment]
> The second example is completely wrong both assumption and output. I assume this is a copy&past error?

We expect only one indentation tab, but we get two. Expected is changed (with `-`) and the result output is added (with `+`).

**Comment by gmaurel** [comment]
Fixed at 0aa4f1863aa6249c9a65d483cc06e27c6bed98c8, Dec 29 19:00:57 2017
